docs/apis/avalanchego/public-api-server.md

```diff
@@ -4,11 +4,18 @@ description: There is a public API server that allows developers to access the A
 
 # Public API Server
 
-There is a public API server that allows developers to access the Avalanche network without having to run a node themselves. The public API server is actually several [AvalancheGo](https://github.com/ava-labs/avalanchego) nodes behind a load balancer to ensure high availability and high request throughput.
+There is a public API server that allows developers to access the Avalanche
+network without having to run a node themselves. The public API server is
+actually several [AvalancheGo](https://github.com/ava-labs/avalanchego) nodes
+behind a load balancer to ensure high availability and high request throughput.
 
-## Using the Public API nodes
+## Using the Public API Nodes
```

Comment:
You should link out to Subnet APIs too.

Reply:
Done https://github.com/ava-labs/avalanche-docs/pull/1098/commits/b1fef276a24ab9b8889801c61be7d871a905eefb